pkg/graveler/hooks_handler.go

```diff
@@ -23,17 +23,15 @@ const (
 	EventTypePreDeleteBranch  EventType = "pre-delete-branch"
 	EventTypePostDeleteBranch EventType = "post-delete-branch"
 
-	RunIDTimeLayout = "20060102150405"
-	UnixYear3000    = 32500915200
+	UnixYear3000 = 32500915200
 )
 
 // HookRecord is an aggregation of all necessary fields for all event types
 type HookRecord struct {
 	// Required fields for all event types:
-	RunID            string
-	EventType        EventType
-	RepositoryID     RepositoryID
-	StorageNamespace StorageNamespace
+	RunID      string
+	EventType  EventType
+	Repository *RepositoryRecord
```

Comment:
`RepositoryID` was concrete, and `RepositoryRecord` is a reference.
Is it for not passing the whole object, or is there a different reason for this?

Reply:
yes